src/applications/pensions/config/chapters/02-military-history/servicePeriod.js

```diff
@@ -34,11 +34,12 @@ export default {
       labels: serviceBranchLabels,
       required: true,
     }),
-    activeServiceDateRange: dateRangeUI(
-      'Date initially entered active duty',
-      'Final release date from active duty',
-      'Date initially entered active duty must be before final date released from active duty',
-    ),
+    activeServiceDateRange: currentOrPastDateRangeUI({
+      fromOptions: 'Date initially entered active duty',
```

Comment:
oh, now seeing how this looks from the consumer, I'm realizing that naming of 'options' is not very intuitive after all...
with other patterns we call it `options`, but the usage of that is: pass a string or object directly rather than assigning it to something called 'options'. hmm...

e.g. most patterns we do:
patternUI('name of field') or
patternUI({
   title: 'name of field',
   hint: 'custom hint'
   etc.. any ui:option...
})

we allow the simple version or the complex version for patterns.

I actually think the code before, how dateRangeUI was used on the left diff, is how I envisioned a simple pattern could work, and would be in line with our other patterns.
What do you think of..

currentOrPastDateRangeUI('field name from', 'field name to', 'optional error message')  for the simple use case, and if they want more advanced, then they pass an object:
currentOrPastDateRangeUI({
   fromLabel: ...
   toLabel ...
   other properties
})
and we keep your approach of using fromLabel, toLabel ?

Reply:
I've pushed an update to accommodate these patterns for simple and advanced usage, where advanced accepts an object for each date.

```
* // Simple usage:
 * exampleDateRange: currentOrPastDateRangeUI(
 *   'Start date of event',
 *   'End date of event',
 *   'Custom error message'
 * )
 *
 * // Advanced usage:
 * exampleDateRange: currentOrPastDateRangeUI(
 *   { title: 'Start date of event', ... },
 *   { title: 'End date of event', hint: 'This is a hint' },
 *   'Custom error message'
 * })
 ```